local-flake/flake.nix

```diff
@@ -58,6 +58,30 @@
           ldflags = ["-s" "-w" "-X main.version=${version}"];
         };
 
+        cloud-mta-build-tool = nixpkgsFor.${system}.buildGoModule rec {
+          pname = "Cloud MTA Build Tool";
+          version = "1.2.26";
```

Comment:
Do we have any mechanism so far that bumps this?

Reply:
No, but I think @joergdw was trialing some CI in the context of another repo ...